service/src/test/java/uk/gov/hmcts/reform/fpl/service/CourtLevelAllocationServiceTest.java

```diff
@@ -0,0 +1,116 @@
+package uk.gov.hmcts.reform.fpl.service;
```

Comment:
could we just get some whens in the tests titles in this file. 

shouldAddYesToMissingAllocationDecision doesn't tell me what I need to do to addYes. 

shouldAddAllocationPresentYesWhenAllocationDecisionIsMissing? Although present = yes when missing?? 🤔